onnx/defs/traditionalml/defs.cc

```diff
@@ -314,103 +314,147 @@ static const char* LabelEncoder_ver2_doc = R"DOC(
     "Sally"] would be mapped to [-1, 5, 5, 6, 6].<br>
     Since this operator is an one-to-one mapping, its input and output shapes
     are the same. Notice that only one of 'keys_*'/'values_*' can be set.<br>
+    When 'keys_as_tensor' and 'values_as_tensor' are used, the default value
```

Comment:
Do we need this constraint that if one is specified as a tensor that the other must be also specified as a tensor? (Among keys/values/default.)  In a sense it doesn't matter much, since a backend needs to support all the variations as it is. It seems simpler to avoid extra-constraints (unless there is a good reason).

Reply:
Good point. My original plan was to allow keys_as_tensor and values_as_tensor to be of any shape (as long as their shapes were identical, and in that set up it wouldn't make sense to mix *_as_tensor attributes with others. Now we force these attributes to be 1D there's no reason to have this.